subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractIntegrationSpec.groovy

```diff
@@ -94,14 +95,27 @@ class AbstractIntegrationSpec extends Specification {
         new GradleContextualExecuter(distribution, temporaryFolder, getBuildContext())
     }
 
+    @CompileStatic
```

Comment:
❓ Are these annotations required now?

Reply:
No, it just makes the runtime a bit faster when you apply them. No other reason. Just a simple performance improvement I did while I was in there.

Reply:
Some of those changes aren't necessary. Like the added `@CompileStatic(TypeCheckingMode.SKIP)` to the unchanged `runAndFail` function below. I think it should be reviewed and fixed, or, dropped from this PR.

I would also prefer `@CompileStatic` to be added to the type with selected `SKIP` on functions where it makes sense instead of adding it to each function in an ad-hoc way.

Reply:
I'd love to go with the review and fix solution (by adding it to the type and then selectively disabling it), but if that's too complicated, I'm happy to just drop them entirely.

Reply:
Thanks, @JLLeitschuh. Better to drop them from this PR then.

Reply:
@bamboo Done, they are also removed from the methods where I had them before.